frontend/src/components/CompareModels.vue

```diff
@@ -36,6 +36,20 @@
       </div>
       <br />
       <br />
+      <h3 class="title is-size-3">Historical comparisions</h3>
+      <div>
+        <b>Human-GEM</b> was created by unifying the two model lineages
```

Comment:
While 2 lineages are the primary ones, there is also a third as indicated in Fig S1 in the [supplementary file](https://www.science.org/action/downloadSupplement?doi=10.1126%2Fscisignal.aaz1482&file=aaz1482_sm.pdf)

```suggestion
        <b>Human-GEM</b> was created by unifying several model lineages, including
```